src/components/TaskListItem/TaskListItem.less

```diff
@@ -60,13 +60,13 @@ input.task-list-item-checkbox[type="checkbox"] {
 .task-list-item-title {
 }
 
-.octicon-issue-closed,
-.git-pull-request.closed {
+.octicon-issue.closed,
```

Comment:
Hmm, it looks like the class is still `octicon-issue-closed` or `octicon-issue-open`, with an additional `closed` or `opened`, based on

```javascript
iconClasses.push(`octicon-issue-${state}`)
```